x-pack/plugins/reporting/public/index.ts

```diff
@@ -18,12 +17,6 @@ export interface ReportingSetup {
    *
    * @returns boolean
    */
-  usesUiCapabilities: () => boolean;
-
-  /**
-   * A set of React components for displaying a Reporting share menu in an application
-   */
-  components: ReportingPublicComponents;
```

Comment:
We still need this in Reporting for supporting the report-generation menus in Canvas: See `x-pack/plugins/canvas/public/services/kibana/reporting.ts`